src/components/FormComponent/FormComponent.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import InputFieldComponent from '../InputFieldComponent/InputFieldComponent'
+
+class FormComponent extends Component {
+	constructor() {
+		super();
+		this.state = {
+			checkbox: false,
+			email: '',
+			firstName: '',
+			lastName: '',
+			password: '',
+		}	
+	}
+
+	handleInput = (e) => {
+		const target = e.target;
+	    const value = target.type === 'checkbox' ? target.checked : target.value;
+	    const name = target.name;
+
+	    console.log("Name: " + name);
+
+	    this.setState({
+	    	[name]: value
+	    });
+
+
+	}
+
+	handleSubmit = (e)  => {
+		e.preventDefault();
```

Comment:
Lets use event instead of e and make the code more readable

Reply:
Both are acceptable and widely used, but event indeed is more appropriate